utils/model_runners/costwrapper.py

```diff
@@ -0,0 +1,303 @@
+import boto3
+import json
+import traceback
+import sys
+import os
+
+client = boto3.client('pricing')
+
+modelNameMap = {}
+for model in boto3.client('bedrock').list_foundation_models()['modelSummaries']:
+    modelNameMap[model['modelId']] = model['modelName']
+
+client = boto3.client('pricing', region_name = 'us-east-1')
```

Comment:
the pricing client requires the region because the pricing api is only available in few regions. 

it's then using the implicit region from the current region in the user profile later on:
for instance filter:
      {"Field": "regionCode", "Value": boto3.session.Session().region_name , "Type": "TERM_MATCH"},
for Bedrock price list that gets retrieved whole:
            if region_code != boto3.session.Session().region_name:
                continue